config/config.exs

```diff
@@ -70,17 +70,19 @@ config :spacesuit, session_service: %{ enabled: false }
 # Configuration for the CORS middleware
 config :spacesuit, cors: %{
   # Main kill switch to disable the middleware
-  enabled: false, 
-  # Required prefix match in order to process CORS
-  #path_prefixes: ["/matched"],
-  # Maximum age for preflight requests
+  enabled: false,
+  # The path prefixes to filter.
```

Comment:
This description actually confused me. The intent is, 'any requests which match this prefix should be processed as CORS requests', correct?